### PR TITLE
Fix hanami destroy app description

### PR DESCRIPTION
**Pull request by profx5**

### CODE REVIEW

**Comment by profx5** [comment]
http://hanamirb.org/guides/head/command-line/destroy/

File: lib/hanami/cli_sub_commands/destroy.rb [hunk 1]
```diff
@@ -76,11 +76,11 @@ def model(name)
         end
       end
 
-      desc 'application NAME', 'Destroy an application'
+      desc 'app NAME', 'Destroy an application'
```
Comment (davydovanton) [inline lib/hanami/cli_sub_commands/destroy.rb]: maybe is better change to `application_name`?

**Comment by cllns** [comment]
I wonder if we should update this documentation instead of this? 

Should add this as an alias, rather than changing it? Changing the command would be a breaking change

**Comment by profx5** [comment]
Ok, add alias is a good idea. 

It would be good that destroy command would have the same options as a generate command.

**Comment by AlfonsoUceda** [comment]
@profx5 thanks for the fix but it isn't needed. CLI was rewrote with gem [hanami-cli](https://github.com/hanami/cli). You can check the code in develop branch: https://github.com/hanami/hanami/tree/develop/lib/hanami/cli/commands

**Comment by jodosha** [comment]
@profx5 I want to inform you that we have fixed this in Hanami v1.1.0:

```shell
➜ bundle exec hanami version
v1.1.0.beta3

➜ bundle exec hanami destroy app --help
Command:
  hanami destroy app

Usage:
  hanami destroy app APP

Description:
  Destroy an app

Arguments:
  APP                 	# REQUIRED The application name (eg. `web`)

Options:
  --help, -h                      	# Print this help

Examples:
  hanami destroy app admin # Destroy `admin` app
```

**Comment by profx5** [comment]
@jodosha Awesome!

**Pull request closed without merging**